identity-wallet/src/state/extensions/mod.rs

```diff
@@ -0,0 +1,17 @@
+use super::FeatTrait;
+use serde::{Deserialize, Serialize};
+use ts_rs::TS;
+
+
+pub mod actions;
+pub mod reducers;
+
+// This module is soly for testing and demonstrating the extension system.
+#[derive(Debug, Serialize, Deserialize, TS, PartialEq, Default, Clone)]
+pub struct CustomExtension {
+    pub name: String,
+    pub value: String,
+}
+
+#[typetag::serde(name = "customextension")]
```

Comment:
Probably a bit nit-picky but I'd say best to follow conventional use of snake_case here: `custom_extension`. (If you agree this should also be changed in `unime/src-tauri/tests/fixtures/states/test_extension.json`)